edit/edit.js

```diff
@@ -0,0 +1,2068 @@
+      CM.keyMap.pcDefault['F3'] = 'findNext';
+    }
+    if (!extraKeysCommands.findPrev) {
+      CM.keyMap.pcDefault['Shift-F3'] = 'findPrev';
+    }
+
+    // try to remap non-interceptable Ctrl-(Shift-)N/T/W hotkeys
+    ['N', 'T', 'W'].forEach(function(char) {
+      [{from: 'Ctrl-', to: ['Alt-', 'Ctrl-Alt-']},
+       {from: 'Shift-Ctrl-', to: ['Ctrl-Alt-', 'Shift-Ctrl-Alt-']} // Note: modifier order in CM is S-C-A
+      ].forEach(function(remap) {
+        const oldKey = remap.from + char;
+        Object.keys(CM.keyMap).forEach(function(keyMapName) {
+          const keyMap = CM.keyMap[keyMapName];
+          const command = keyMap[oldKey];
+          if (!command) {
+            return;
+          }
+          remap.to.some(function(newMod) {
+            const newKey = newMod + char;
+            if (!(newKey in keyMap)) {
+              delete keyMap[oldKey];
+              keyMap[newKey] = command;
+              return true;
+            }
+          });
+        });
+      });
+    });
+  }
+
+  // user option values
+  CM.getOption = function(o) {
+    return CodeMirror.defaults[o];
+  };
+  CM.setOption = function(o, v) {
+    CodeMirror.defaults[o] = v;
+    editors.forEach(function(editor) {
+      editor.setOption(o, v);
+    });
+  };
+
+  CM.prototype.getSection = function() {
+    return this.display.wrapper.parentNode;
+  };
+
+  // initialize global editor controls
+  function optionsHtmlFromArray(options) {
+    return options.map(function(opt) { return '<option>' + opt + '</option>'; }).join('');
+  }
+  const themeControl = document.getElementById('editor.theme');
+  const themeList = localStorage.codeMirrorThemes;
+  if (themeList) {
+    themeControl.innerHTML = optionsHtmlFromArray(themeList.split(/\s+/));
+  } else {
+    // Chrome is starting up and shows our edit.html, but the background page isn't loaded yet
+    const theme = prefs.get('editor.theme');
+    themeControl.innerHTML = optionsHtmlFromArray([theme == 'default' ? t('defaultTheme') : theme]);
+    getCodeMirrorThemes().then(() => {
+      const themes = (localStorage.codeMirrorThemes || '').split(/\s+/);
+      themeControl.innerHTML = optionsHtmlFromArray(themes);
+      themeControl.selectedIndex = Math.max(0, themes.indexOf(theme));
+    });
+  }
+  document.getElementById('editor.keyMap').innerHTML = optionsHtmlFromArray(Object.keys(CM.keyMap).sort());
+  document.getElementById('options').addEventListener('change', acmeEventListener, false);
+  setupLivePrefs();
+
+  hotkeyRerouter.setState(true);
+}
+
+function acmeEventListener(event) {
+  const el = event.target;
+  const option = el.id.replace(/^editor\./, '');
+  //console.log('acmeEventListener heard %s on %s', event.type, el.id);
+  if (!option) {
+    console.error('acmeEventListener: no "cm_option" %O', el);
+    return;
+  }
+  let value = el.type == 'checkbox' ? el.checked : el.value;
+  let url;
+  const themeLink = document.getElementById('cm-theme');
```

Comment:
I disagree. Just because eslint warns about block scope of let/const doesn't mean it's semantically correct to extract the theme-specific code outside of its case. Either keep `var` or add a eslint override comment